results/app.js

```diff
@@ -0,0 +1,7 @@
+
+const homeButton = document.getElementById('home-button');
+
+
+homeButton.addEventListener('click', () => {
+    window.location.href = '../index.html';
```

Comment:
You also need to clear local storage in this eventListener to start a new game fresh